src/com/pentalog/pentastagiu/homework2/BankAccount.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class BankAccount {
+    private int id;
```

Comment:
daca iti dai seama ca nu te folosesti de un field, cel mai bine e sa il stergi, nu are rost sa poluezi codul pentru ca te poate induce in eroare